site/content/en/latest/user/oidc.md

```diff
@@ -90,33 +123,34 @@ kubectl get securitypolicy/oidc-example -o yaml
 
 ## Testing
 
-Port forward gateway 443 port to localhost:
+Port forward gateway port to localhost:
 
 ```shell
 export ENVOY_SERVICE=$(kubectl get svc -n envoy-gateway-system --selector=gateway.envoyproxy.io/owning-gateway-namespace=default,gateway.envoyproxy.io/owning-gateway-name=eg -o jsonpath='{.items[0].metadata.name}')
 
-sudo kubectl -n envoy-gateway-system port-forward service/${ENVOY_SERVICE} 443:443
+kubectl -n envoy-gateway-system port-forward service/${ENVOY_SERVICE} 8080:80
 ```
 
-Put www.example.com in the /etc/hosts file in your test machine, so we can use this host name to access the demo from a browser:
+Put www.example.com in the /etc/hosts file in your test machine, so we can use this host name to access the gateway from a browser:
 
 ```shell
 ...
 127.0.0.1 www.example.com
 ```
 
-Open a browser and navigate to the `https://www.example.com` address. You should be redirected to the Google login page. After you
-successfully login, you should see the response from the backend service.
+Open a browser and navigate to the `http://www.example.com:8080/myapp` address. You should be redirected to the Google 
+login page. After you successfully login, you should see the response from the backend service.
 
 ## Clean-Up
 
 Follow the steps from the [Quickstart](../quickstart) guide to uninstall Envoy Gateway and the example manifest.
 
-Delete the SecurityPolicy and the secret:
+Delete the SecurityPolicy, the secret, and the HTTPRoute:
```

Comment:
It should become like this: `...., the secret and the HttpRoute`

Reply:
Not a native speaker, but AFAIK, both are grammatically correct?